src/pybamm_cookiecutter/_version.py

```diff
@@ -0,0 +1,16 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
+TYPE_CHECKING = False
+if TYPE_CHECKING:
+    from typing import Tuple, Union
+    VERSION_TUPLE = Tuple[Union[int, str], ...]
+else:
+    VERSION_TUPLE = object
+
+version: str
+__version__: str
+__version_tuple__: VERSION_TUPLE
+version_tuple: VERSION_TUPLE
+
+__version__ = version = '0.1.dev142+g0fc714a.d20240813'
+__version_tuple__ = version_tuple = (0, 1, 'dev142', 'g0fc714a.d20240813')
```

Comment:
If the file isn't to be tracked in version control (as mentioned by the setuptools_scm auto-generated comment here), we should find another method to handle this.

Reply:
I am not sure how we would go about handling this, maybe we should remove `hatch-vcs` from the project as we are anyways going to be following semantic versioning.

Reply:
Just saw this. Semantic/calendar versioning choice is different from VCS/manual versioning choice. VCS versioning just makes our life easy. I am not sure why is this file required to be tracked by `git` in this PR, but we can discuss this in today's meeting!

Reply:
Makes sense, I am removing the file for now